forms-flow-theme/scss/_modal.scss

```diff
@@ -265,65 +271,77 @@
     min-height: 4.5rem;
     gap: var(--spacer-050);
     cursor: pointer;
-    .template-desc{
+
+    .template-desc {
       display: -webkit-box;
-      -webkit-line-clamp: 2; /* Show 2 lines */
+      -webkit-line-clamp: 2;
+      /* Show 2 lines */
       -webkit-box-orient: vertical;
       overflow: hidden;
       text-overflow: ellipsis;
-      white-space: normal; 
+      white-space: normal;
     }
   }
+
   .active-template {
     border-bottom: 1px solid var(--ff-gray-000, #f2f2f3);
     background: var(--ff-base-100, #E7E9FE);
     max-height: none !important;
+
     .template-desc {
-      display: block; 
+      display: block;
       -webkit-line-clamp: unset;
       overflow: visible;
       white-space: normal;
     }
   }
-  .template-title ,.template-desc {
+
+  .template-title,
+  .template-desc {
     color: var(--ff-dark, #000);
     font-size: var(--font-size-xs);
     font-weight: var(--font-weight-sm);
-    line-height: var( --text-line-height);  
+    line-height: var(--text-line-height);
   }
-  .form-container{
-    overflow-y: scroll; 
-    -ms-overflow-style: none; 
+
+  .form-container {
+    overflow-y: scroll;
+    -ms-overflow-style: none;
     scrollbar-width: none;
```

Comment:
please remove this @Bonymol-aot  . Alex will be adding this property as a generic one so that scrollbar wont be visible across application